repairnator/build-rainer/src/main/java/fr/inria/repairnator/Launcher.java

```diff
@@ -0,0 +1,8 @@
+package fr.inria.repairnator;
+
+public class Launcher {
+    public static void main( String[] args ) {
```

Comment:
put the main in class BuildRainer directly

Reply:
Fixed.